core/app/c/[communitySlug]/pubs/[pubId]/page.tsx

```diff
@@ -232,13 +232,6 @@ export default async function Page(props: {
 			</div>
 			<div>
 				<h2 className="text-xl font-bold">Pub Contents</h2>
-				<p className="text-muted-foreground">
-					Use the "Add New Pub" button below to create a new pub and add it to this pub's
-					contents.
-				</p>
-			</div>
-			<div className="mb-2">
-				<CreatePubButton text="Add New Pub" communityId={community.id} parentId={pub.id} />
```

Comment:
I know this ticket wasn't to remove the add children button, but since it's getting removed anyway I did it now so that `CreatePubButton` didn't have to keep working with children